### PR TITLE
Feature/dsi 5629 invite journey auditlog

### PR DESCRIPTION
**Pull request by malikimran34**

### CODE REVIEW

File: src/infrastructure/logger/index.js [hunk 1]
```diff
@@ -26,14 +21,14 @@ const customLevels = {
...
 };
-
 const loggerConfig = {
   levels: customLevels.levels,
   transports: [],
 };
-
-// loggerConfig.transports.push(new winston.transports.Console({ level: logLevel, colorize: true }));
-
+const sequelizeTransport = WinstonSequelizeTransport(config);
```
Comment (ipredoi) [inline src/infrastructure/logger/index.js:28]: Is there any particular reason to replace  `AuditTransporter` with `WinstonSequelizeTransport`?

Comment (malikimran34) [reply src/infrastructure/logger/index.js:28]: Logging to database was not working, the package is missing as used in Profile and other components login.dfe.audit.winston-sequelize-transport

Comment (ipredoi) [reply src/infrastructure/logger/index.js:28]: Services component is logging correctly to Prod DB and looks like there is an infrastructure problem not publishing to the lower envs DBs, which needs to be investigated. 
`AuditTransporter` and and queues should be used everywhere and all the instances where `WinstonSequelizeTransport` is used should be replaced.

**Review by ipredoi — CHANGES_REQUESTED** [review]
Please revert the changes made into the `infrastructure/loger/index.js` file

**Pull request closed without merging**